apps/base-docs/tutorials/docs/5_shopify-storefront-commerce.md

```diff
@@ -0,0 +1,99 @@
+---
+title: 'Deploy a Shopify Storefront with Coinbase Commerce'
+slug: /shopify-storefront-commerce
+description: Learn how to launch a Shopify storefront that uses Coinbase Commerce as a crypto payment gateway.
+author: hughescoin
+keywords:
+  [Shopify, Coinbase Commerce, crypto payments, ecommerce, Hydrogen, Oxygen, USDC, ocs, onchain]
+tags: ['ecommerce', 'crypto', 'shopify', 'onchain summer', ocs]
+difficulty: easy
+hide_table_of_contents: false
+displayed_sidebar: null
+---
+
+Deploy a Shopify Storefront with Coinbase Commerce Plugin
+[Shopify](https://www.shopify.com/) provides a robust platform for creating online stores. This tutorial will guide you through setting up a Shopify storefront that integrates with Coinbase Commerce, allowing you to accept cryptocurrency payments.
+
+## Objectives
+
+By the end of this tutorial, you will have a custom storefront with integrated crypto payments, using your existing Shopify products.
```

Comment:
"Have" is usually not a great objective as we're trying to help developers build their toolkit.